### PR TITLE
WIll allow FIO to run test with pvcvolumemode as Block

### PR DESCRIPTION
**Pull request by shekharberry**
### Description
This PR will allow FIO to run tests with pvcblockmode as Block. 

### Fixes
https://github.com/cloud-bulldozer/benchmark-operator/issues/793

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/cloud-bulldozer/benchmark-operator
-  newTag: latest
+  newName: quay.io/shekharberry/bo
```
Comment (jtaleric) [inline config/manager/kustomization.yaml:15]: Need to revert.

File: roles/fio_distributed/tasks/main.yml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 - block:
   - name: Provide path for fio - PV
     set_fact:
-      fio_path: /mnt/pvc
-    when: workload_args.storageclass is defined
+      fio_path: /dev/xvda
```
Comment (jtaleric) [inline roles/fio_distributed/tasks/main.yml:5]: So, if we don't use block is this still correct?

Comment (jtaleric) [reply roles/fio_distributed/tasks/main.yml:5]: meaning should we look at the type of storageclass and if it isn't block we keep the default to `/mnt/pvc` ?

**Review by jtaleric — CHANGES_REQUESTED** [review]
question about always defaulting to `/dev/xvda` as well as removing the personal operator path.

File: roles/fio_distributed/tasks/main.yml (hunk 2 above)
Comment (shekharberry) [reply roles/fio_distributed/tasks/main.yml:5]: It worked for all 3 cases, pvcvolumemode is undefined, Block or Filesystem. When It's Filesystem /dev/xvda is considered as directory by FIO and for block /dev/xvda is considered as location of Block device.

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (shekharberry) [reply config/manager/kustomization.yaml]: Done

**Comment by shekharberry** [comment]
Yes. This fix works and it would be great if someone can verify the same on
their setup so that the PR is merged.

--Shekhar

**Review by jtaleric — APPROVED** [review]
/lgtm

**Comment by jtaleric** [comment]
@shekharberry did you share this PR with the ODF/OCS Team to ensure it doesn't break any of their automation?

**Pull request merged**